### ISSUE TITLE
Patch for /trunk/AndorsTrail/res/values-it/questlist.xml

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#183]
```
italian revision, completed. 
has in issue 182, missing parts, can i add?

Giulio
```

Original issue reported on code.google.com by `liogiu@gmail.com` on 11 Apr 2011 at 8:31

Attachments:
- [questlist.xml.patch](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-183/comment-0/questlist.xml.patch)

**Comment by GoogleCodeExporter** [comment]
```
Thank you again liogiu!

Committed as part of r70.

Yes, you can add the parts that are missing if you want to complete the 
translation.
```

Original comment by `oskar.wi...@gmail.com` on 14 Apr 2011 at 8:35
- Changed state: **Ready**
- Added labels: **Type-Enhancement**, **Milestone-0.6.9**
- Removed labels: **Type-Patch**

**Comment by GoogleCodeExporter** [comment]
```
You are welcome
Ok, I will add the missing parts.

When you are planning to release the 0.6.9 version?

```

Original comment by `liogiu@gmail.com` on 15 Apr 2011 at 8:44

**Comment by GoogleCodeExporter** [comment]
Original comment by `oskar.wi...@gmail.com` on 4 May 2011 at 5:41
- Added labels: **Type-Translation**
- Removed labels: **Type-Enhancement**

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.9 . Closing.
```

Original comment by `oskar.wi...@gmail.com` on 29 Jun 2011 at 7:52
- Changed state: **Done**
